src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/ad/HiviUap.java

```diff
@@ -0,0 +1,87 @@
+package com.wikia.webdriver.pageobjectsfactory.componentobject.ad;
+
+import com.wikia.webdriver.common.core.WikiaWebDriver;
+import com.wikia.webdriver.common.core.elemnt.Wait;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+
+public class HiviUap {
+  private By AD_IFRAME;
+  private By FULL_SCREEN_BUTTON;
+  private By IMA_IFRAME;
+  private By PAUSE_BUTTON;
+  private By REPLAY_BUTTON;
+  private By SOUND_BUTTON;
+  private By VIDEO_CLICK_AREA;
+  private By VIDEO_ELEMENT;
+  private By VIDEO_PLAYER;
+  private By VIDEO_THUMBNAIL;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename this field "VIDEO_THUMBNAIL" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS00116)